### PR TITLE
Feature/#86 fill in the starting locations new2

### PR DESCRIPTION
**Pull request by cml1010101**

### CODE REVIEW

**Review by FRCTeam172 — APPROVED** [review]

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -102,17 +102,15 @@ public final class Constants {
         16.4846, 
         8.1026
     );
-    /** TODO */
```
Comment (tagnihotri) [inline src/main/java/frc/robot/Constants.java:105]: Add comments here in order to explain that positions inputted (numbers) into Pose2d. For example, where on the field is this: (1.693, 4.402, Rotation2d.fromDegrees(180))

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -102,17 +102,15 @@ public final class Constants {
...
     public static final Pose2d[] BLUE_POSES = new Pose2d[] {
-        new Pose2d(),
-        new Pose2d(),
-        new Pose2d()
+        new Pose2d(1.693, 4.402, Rotation2d.fromDegrees(180)),
+        new Pose2d(1.693, 2.761, Rotation2d.fromDegrees(180)),
+        new Pose2d(1.693, 1.083, Rotation2d.fromDegrees(180))
     };
-    /** TODO */
```
Comment (tagnihotri) [inline src/main/java/frc/robot/Constants.java:111]: Add comments here in order to explain that positions inputted (numbers) into Pose2d. For example, where on the field is this:(14.895, 1.083, Rotation2d.fromDegrees(0))

File: src/main/java/frc/robot/Constants.java [hunk 3]
```diff
@@ -102,17 +102,15 @@ public final class Constants {
...
     public static final Pose2d[] RED_POSES = new Pose2d[] {
-        new Pose2d(),
-        new Pose2d(),
-        new Pose2d()
+        new Pose2d(14.895, 1.083, Rotation2d.fromDegrees(0)),
+        new Pose2d(14.895, 2.761, Rotation2d.fromDegrees(0)),
+        new Pose2d(14.895, 4.402, Rotation2d.fromDegrees(0))
     };
     /** General Constants */
```
Comment (tagnihotri) [inline src/main/java/frc/robot/Constants.java:115]: Look at changing the comment structure to better explain the comments in the code we look at.

**Review by tagnihotri — COMMENTED** [review]
These look great but explaining them in comments (as if you were explaingin to someone who doesn't know code) would be great)!

**Review by tagnihotri — APPROVED** [review]

**Pull request merged**